submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/common/config/SecurityConfig.kt

```diff
@@ -44,6 +44,10 @@ class SecurityConfig(
             .and()
             .authorizeRequests()
             .antMatchers("/auth/**").permitAll()
+            .antMatchers("/v2/**").permitAll()
```

Comment:
what are this resources starting with `v2` ?

Reply:
This is the path for the swagger resource generated by the annotations, which later is used by the swagger-ui plugin to show the documentation